src/app/retrospective/sprint-list/sprint-list.data-source.ts

```diff
@@ -39,9 +43,6 @@ export class SprintListDataSource extends DataSource<any> {
                 },
                 err => {
                     this.errorCallback(err);
-                },
-                () => {
```

Comment:
The complete callback is not fired if the error callback is fired (Refer https://github.com/angular/angular/issues/7865)